IPython/terminal/shortcuts.py

```diff
@@ -68,9 +68,8 @@ def create_ipython_shortcuts(shell):
                           & insert_mode
                           & cursor_in_leading_ws
                         ))(indent_buffer)
-
     kb.add('c-o', filter=(has_focus(DEFAULT_BUFFER)
-                  & emacs_insert_mode))(newline_autoindent_outer(shell.input_splitter))
+                  & emacs_insert_mode))(newline_autoindent_outer(shell.input_transformer_manager))
```

Comment:
A comment on style here. Perhaps going to a longer line length would improve readability and understanding. i.e. line 74 is more understandable than 71-72 and 66-70